application/apache-kafka-flink-streaming/README.md

```diff
@@ -40,7 +40,7 @@ into your local `.env` file.
 ### Run the docker compose (and build the images)
 
 ```
-docker compose up -d --build
+docker-compose up -d --build
```

Comment:
That would use an older Docker Compose I guess. The new one is part of the Docker command itself.

-- https://docs.docker.com/compose/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2

Reply:
Let's stick with the previous incantation, when possible?